### PR TITLE
feat: add fetch by country at flixhq.ts

### LINKED ISSUE #467: Add country route to flixHQ provider
**Is your feature request related to a problem? Please describe.**
I would like to have the country route option to get movies and tv series by country.

**Describe the solution you'd like**
add the scraper for the route country so when the person type /?country=KR
This route should bring movies and tv series filtered by that country.

### PR DESCRIPTION
**Pull request by dhelbegor**
**What kind of change does this PR introduce?**
This change will add the possibility to fetch movies/tvseries by country at flixhq.

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
Not yet

**Other information**
Closes #467

### CODE REVIEW

File: src/models/types.ts [hunk 1]
```diff
@@ -43,6 +43,10 @@ export interface ISearch<T> {
   results: T[];
 }
 
+export interface ISearch<T> extends Pagination<T> {}
+
+export interface IByCountry<T> extends Pagination<T> {}
```
Comment (riimuru) [inline src/models/types.ts:48]: Isn't `UByCountry<T>` interface the same as using `ISearch<T>`? I don't think its necessery to have these extra empty interfaces

Comment (riimuru) [reply src/models/types.ts:48]: Everything else looks good! 👍

Comment (riimuru) [reply src/models/types.ts:48]: @dhelbegor

Comment (dhelbegor) [reply src/models/types.ts:48]: Sorry for the delay in responding you @riimuru . I took the weekend off to travel.

My intention creating a ISearch and IByCountry was to have separete responsibilies. so one will no affect the other in the future. I will change and do a commit right now :D.

Comment (dhelbegor) [reply src/models/types.ts]: Done @riimuru  :D. thank u for your review

Comment (riimuru) [reply src/models/types.ts]: Thank you!

**Review by riimuru — APPROVED** [review]

**Pull request merged**